doc/source/whatsnew/v0.23.0.txt

```diff
@@ -965,6 +965,8 @@ Plotting
 ^^^^^^^^
 
 - :func:`DataFrame.plot` now raises a ``ValueError`` when the ``x`` or ``y`` argument is improperly formed (:issue:`18671`)
+- :func:`DataFrame.plot` now supports multiple columns to the ``y`` argument (:issue:`19699`)
```

Comment:
this first should be on Other Enhancements section

Reply:
Could you move this to other enhancements?